src/openrct2/management/Research.cpp

```diff
@@ -175,7 +175,7 @@ static void research_next_design()
         it++;
     }
 
-    gResearchNextItem = researchItem;
+    gResearchNextItem = std::make_optional(researchItem);
```

Comment:
`std::optional` is not require here as you are setting an optional field to an actual value of a ResearchItem. It is like doing:

```
std::optional<int> x = std::make_optional(5);
```
rather than just:
```
std::optional<int> x = 5;
```